backend/company-agent/src/main/resources/databasemigrations/V1.3__add-bpaschema-read-only.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE bpaschema ADD COLUMN is_read_only boolean;
```

Comment:
Broken. You need to add a new file.

Reply:
what do you mean with new file? can you give an example?

Reply:
sorry my bad, this is a new file. ignore this comment.